ceph/cephfs/cephfs_provisioner/cephfs_provisioner.py

```diff
@@ -196,6 +196,15 @@ def cap_update(orig, want, unwanted):
         assert caps[0]['entity'] == client_entity
         return caps[0]
 
+    def _remove_ceph_user(self, user_id):
+        client_entity = "client.{0}".format(user_id)
+        ret = self._volume_client._rados_command(
+            'auth rm',
```

Comment:
interesting, `auth rm` works but according to [doc](http://docs.ceph.com/docs/jewel/rados/operations/user-management/), should be `auth del`